src/packages/@ncigdc/tableModels/utils.tsx

```diff
@@ -83,36 +96,55 @@ export const createDataCategoryColumns = ({
           >
             {_.sumBy(
               hits.edges,
-              x =>
+              (x: any) =>
                 findDataCategory(category.abbr, x.node.summary.data_categories)[
                   countKey
-                ],
+                ]
             ).toLocaleString()}
           </Link>
         </TdNum>
       ),
     })),
   ];
 };
-
-type TCreateSelectColumn = ({
-  idField: string,
-  headerRowSpan?: number,
-}) => {};
-export const createSelectColumn: TCreateSelectColumn = ({
+export interface IColumnProps<NoTH> {
+  name: string;
+  id: string;
+  sortable: boolean;
+  downloadable: boolean;
+  hidden: boolean;
+  field?: string;
+  subHeading?: boolean;
+  subHeadingIds?: NoTH extends true? string[] : undefined;
+  parent?: string;
+  th: (props: any) => JSX.Element;
+  td: NoTH extends true? undefined : (props: any) => JSX.Element;
```

Comment:
src/packages/@ncigdc/modern_components/CancerDistributionTable/CancerDistributionTable.model.tsx

`td: TNode;`

src/packages/@ncigdc/modern_components/CancerDistributionTable/types.ts

```
export interface INode {
  project_id: string;
  disease_type?: any;
  site?: any;
  num_affected_cases?: any;
  num_affected_cases_total?: any;
  num_affected_cases_percent?: any;
  num_cnv_gain?: any;
  num_cnv_gain_percent?: any;
  num_cnv_loss?: any;
  num_cnv_loss_percent?: any;
  num_cnv_cases_total?: any;
}
```

Ironically I have a todo about removing all the any types ...